### PR TITLE
Return datasets instead of wrappers

### PR DESCRIPTION
**Pull request by stefanoborini**
The run method now returns the datasets instead of the wrappers. The wrappers are now purely internal to the run

### CODE REVIEW

File: simphony_ui/ui.py [hunk 1]
```diff
@@ -152,56 +151,33 @@ def run_calc(self):
         future = self._executor.submit(self._run_calc_threaded)
         future.add_done_callback(self._calculation_done)
 
-    @on_trait_change('openfoam_wrapper')
-    def show_openfoam_result(self):
+    @on_trait_change('datasets')
+    def update_sources(self):
         """ Function which add the Openfoam dataset to the
```
Comment (martinRenou) [inline simphony_ui/ui.py]: This doc is not relevant anymore

File: simphony_ui/ui.py [hunk 2]
```diff
@@ -152,56 +151,33 @@ def run_calc(self):
...
-        self._clear_openfoam_source()
+        self._clear_sources()
 
-        if self.openfoam_wrapper is None:
+        if self.datasets is None:
             return
 
-        mayavi_engine = self.mlab_model.engine
-
         # Get Openfoam dataset
```
Comment (martinRenou) [inline simphony_ui/ui.py]: Same here

File: simphony_ui/ui.py [hunk 3]
```diff
@@ -152,56 +151,33 @@ def run_calc(self):
...
-        openfoam_dataset = self.openfoam_wrapper.get_dataset(
-            self.openfoam_wrapper.get_dataset_names()[0])
-        self.openfoam_source = dataset2cudssource(openfoam_dataset)
+        self.sources = tuple(map(dataset2cudssource, self.datasets))
 
+    @on_trait_change('sources')
+    def show_sources(self):
         # Add Openfoam source
```
Comment (martinRenou) [inline simphony_ui/ui.py]: Same here

File: simphony_ui/ui.py [hunk 4]
```diff
@@ -317,37 +293,23 @@ def reset(self):
         """ Function which reset the Mayavi scene.
         """
         # Clear scene
-        self._clear_openfoam_source()
-        self._clear_liggghts_sources()
+        self._clear_sources()
 
         # Clear wrappers
```
Comment (martinRenou) [inline simphony_ui/ui.py]: Clear datasets

File: simphony_ui/ui.py [hunk 5]
```diff
@@ -298,7 +274,7 @@ def _update_result(self, result):
         self.progress_dialog.update(100)
```
Comment (martinRenou) [inline simphony_ui/ui.py:271]: The docstring is not relevant here

**Review by martinRenou — DISMISSED** [review]

**Comment by stefanoborini** [comment]
@martinRenou should be green

**Pull request merged**